akka-stream/src/main/java-jdk9-only/akka/stream/javadsl/JavaFlowSupport.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http//www.lightbend.com/>
+ */
+
+package akka.stream.javadsl;
+
+import akka.NotUsed;
+import akka.japi.Pair;
+import akka.japi.Creator;
+import akka.stream.impl.JavaFlowAndRsConverters;
```

Comment:
JavaFlowAndRsConverters have been made internal after internal discussion (there will be RS provided adapters, so we don't want to duplicate the effort and confuse users)